pkg/azure/utils.go

```diff
@@ -529,3 +529,28 @@ func DoesVnetOverlapWithParaglider(ctx context.Context, handler *AzureSDKHandler
 
 	return false, nil
 }
+
+// CreateVirtualNetworkParameters creates and returns an instance of armnetwork.VirtualNetwork
+// with the specified parameters.
+//
+// Subnet address space is the same as the Vnet address space.
+func CreateVirtualNetworkParameters(ctx context.Context, location string, vnetName string, addressSpace string) armnetwork.VirtualNetwork {
```

Comment:
Could we rename this to GetVirtualNetworkParameters? Create makes it sound like it's going to create the VNET.

And also, we shouldn't export this method unless there is a major reason to use this in other packages.

Reply:
Alright